app/views/email/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :page_title, 'Your email address' %>
```

Comment:
[Design system says](https://design-system.service.gov.uk/patterns/validation/#how-to-tell-the-user-about-validation-errors):

> If the user’s answers fail validation:
> - add ‘Error: ’ to the beginning of the page <title> so screen readers read it out as soon as possible

So `Error:` should be prefixed here if there are errors... might be worth doing it in the layout or somehow abstracting it so we don't have to do it on every form.

Reply:
👍